deck_of_cards/lib/deck_of_cards.rb

```diff
@@ -0,0 +1,17 @@
+class DeckOfCards
+  attr_reader :deck
+
+  def initialize
+    suit = %w( C D H S )
+    rank = %w( Ace King Queen Jack 10 9 8 7 6 5 4 3 2 )
+    @deck = rank.product(suit).map(&:join)
```

Comment:
Your representation of a Card here is just a string. That misses the 1st user story laid out in the objectives. I think a Card class would make for better OO here.